features/standby_cluster.feature

```diff
@@ -3,10 +3,9 @@ Feature: standby cluster
   Scenario: check replication of a single table in a standby cluster
     Given I start postgres0 without slots sync
     And I create a replication slot postgres1 on postgres0
-    And I create a role replicator on postgres0
     And I start postgres1 in a standby cluster batman1 as a clone of postgres0
     Then postgres1 is a leader of batman1 after 10 seconds
-    And postgres1 role is the standby_leader after 24 seconds
+    And DCS for batman1 has leader=postgres1 after 20 seconds
```

Comment:
https://github.com/zalando/patroni/blob/master/features/custom_bootstrap.feature#L9 -- is already doing exactly what you need